### PR TITLE
fix: disable consumption when there are no active meters

### PR DESCRIPTION
**Pull request by bdraco**

### CODE REVIEW

File: tests/__snapshots__/test_endpoints.ambr [hunk 1]
```diff
@@ -1119,12 +1119,7 @@
         ]),
       }),
     }),
-    'system_consumption': dict({
-      'watt_hours_last_7_days': 4545910,
-      'watt_hours_lifetime': 4545910,
-      'watt_hours_today': 4545910,
-      'watts_now': 3731,
-    }),
```
Comment (bdraco) [inline tests/__snapshots__/test_endpoints.ambr:1127]: This is the one we just added where we expect consumption to not appear

File: tests/__snapshots__/test_endpoints.ambr [hunk 2]
```diff
@@ -4008,12 +3917,7 @@
         ]),
       }),
     }),
-    'system_consumption': dict({
-      'watt_hours_last_7_days': 0,
-      'watt_hours_lifetime': 0,
-      'watt_hours_today': 0,
-      'watts_now': -1,
-    }),
+    'system_consumption': None,
```
Comment (bdraco) [inline tests/__snapshots__/test_endpoints.ambr:3920]: This was an existing system that was obviously reporting garbage

**Comment by bdraco** [comment]
codecov drop is likely impossible branching with our current test fixtures but too risky to remove at this point. We still need a lot more

**Review by dgomes — APPROVED** [review]

**Comment by bdraco** [comment]
thanks

**Pull request merged**